components/scheduler/views/agenda.md

```diff
@@ -39,15 +39,18 @@ The following parameters allow you to configure the Agenda view:
 
 <TelerikScheduler Data="@Appointments" @bind-Date="@StartDate" Width="800px">
     <SchedulerViews>
+         @* Step 1: Enable the Agenda view in the Scheduler *@
```

Comment:
As this is done in a single step and on a single line, you can merge both steps.

```suggestion
         @* Enable the Agenda view and use the 'HideEmptyAgendaDays' option to control whether days without appointments are shown in the Agenda view. *@
```